Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragment.kt

```diff
@@ -207,20 +217,26 @@ class HomeFragment : Fragment(R.layout.home_fragment_layout), AutoInject {
                 HomeFragmentDirections.actionMainFragmentToSubmissionDispatcher()
             )
             HomeFragmentEvents.OpenFAQUrl -> openUrl(getString(R.string.main_about_link))
-            HomeFragmentEvents.GoToRapidTestResultNegativeFragment -> doNavigate(
-                HomeFragmentDirections.actionMainFragmentToSubmissionNegativeAntigenTestResultFragment()
+            is HomeFragmentEvents.GoToRapidTestResultNegativeFragment -> doNavigate(
+                HomeFragmentDirections.actionMainFragmentToSubmissionNegativeAntigenTestResultFragment(event.identifier)
             )
-            is HomeFragmentEvents.ShowDeleteTestDialog -> showRemoveTestDialog(event.type, event.submission)
```

Comment:
`showRemoveTestDialog` was showing a different dialog with red delete button , can you align with UX ?